docs/content/guides/operations/kubernetes/overview/index.md

```diff
@@ -22,7 +22,7 @@ Radius resources, when deployed to a Kubernetes environment, are mapped to one o
 
 | Radius resource                  | Kubernetes object |
 |----------------------------------|-------------------|
-| [`Applications.Core/containers`]({{< ref container-schema >}}) | `apps/Deployment@v1`<br />`core/Service@v1` _(if ports defined)_ |
+| [`Applications.Core/containers`]({{< ref container-schema >}}) | `apps/Deployment@v1`<br /> <br /> `core/Service@v1` _(if ports defined)_ <br /> <br /> The `connections` property allows users to declare a connection between two resources, which Radius uses to inject resource related information into environment variables. This information is stored in a Kubernetes secret object if it is automatically-configured data and a non-secret Kubernetes object if its manually configured data. They are then used to access the respective resource information without having to hard code URIs, connection strings, access keys, or anything that an application code needs to successfully communicate. |
```

Comment:
Can we make this more succinct? Paragraph-style text isn't usually what we include in tables. Ideally we could match what's already there.

Reply:
Yeah with this one I'm honestly a bit stuck on how to best illustrate that as a kubernetes object as well as convey that its only automated date that's encrypted versus manual data being a string.

```
| Radius resource                  | Kubernetes object |
|----------------------------------|-------------------|
| [`Applications.Core/containers`]({{< ref container-schema >}}) | `apps/Deployment@v1`<br />`core/Service@v1` _(if ports defined)_ |
| [`Applications.Core/containers`]({{< ref container-schema >}}) | `apps/Deployment@v1`<br /> <br /> `core/Service@v1` _(if ports defined)_ <br /> <br />  `connections` -> Kubernetes Secret |
```
????